test/specs/basic/ignore-joiners-synonyms-diacritics.js

```diff
@@ -0,0 +1,29 @@
+/*!***************************************************
+ * mark.js
+ * https://github.com/julmot/mark.js
+ * Copyright (c) 2014–2016, Julian Motz
+ * Released under the MIT license https://git.io/vwTVl
+ *****************************************************/
+"use strict";
+describe("basic mark with ignoreJoiners with synonyms and diacritics", function () {
+    var $ctx;
+    beforeEach(function (done) {
+        loadFixtures("basic/ignore-joiners-synonyms-diacritics.html");
+
+        $ctx = $(".basic-ignore-joiners-synonyms-diacritics");
+        new Mark($ctx[0]).mark(["Dolor", "Aliquam", "Sed", "Lorèm ipsum"], {
+            "separateWordSearch": false,
+            "synonyms": {
+                "Sed": "justø",
+                "Dolor": "Ãmet"
+            },
+            "caseSensitive": true,
```

Comment:
Ok, I'm going to rename the two files to `ignore-joiners-case-sensitive-synonyms-diacritics`, is that okay or should I actually remove the case sensitive option?